libraries/core_libs/network/src/tarcap/packets_handlers/votes_sync_packet_handler.cpp

```diff
@@ -23,8 +25,8 @@ void VotesSyncPacketHandler::validatePacketRlpFormat([[maybe_unused]] const Pack
 
 void VotesSyncPacketHandler::process(const PacketData &packet_data, const std::shared_ptr<TaraxaPeer> &peer) {
   const auto next_votes_count = packet_data.rlp_.itemCount();
-  if (next_votes_count == 0) {
-    LOG(log_er_) << "Receive 0 next votes from peer " << packet_data.from_node_id_
+  if (next_votes_count == 0 || next_votes_count > kMaxVotesInPacket) {
```

Comment:
because of this we need to mark it as breaking-change

Reply:
Marked